scipy/stats/_multivariate.py

```diff
@@ -2620,3 +2621,295 @@ def entropy(self):
     method_frozen.__doc__ = doccer.docformat(
         method.__doc__, wishart_docdict_noparams)
     method.__doc__ = doccer.docformat(method.__doc__, wishart_docdict_params)
+
+
+_doc_multinom_callparam = """\
+n : int
+    Number of trials
+p : array_like, shape (k,)
+    Probabilities. These should sum to one. If they do not, then
+    ``p[-1]`` is modified to account for the remaining probability so that
+    ``sum(p) == 1``.
+"""
```

Comment:
Also, saying that `p[-1]` is modified suggests that we're clobbering input (we're not, are we?)